client/src/pages/Home.js

```diff
@@ -3,11 +3,13 @@ import "./Home.css";
 import { PicCarousel } from "../components/PicCarousel";
 import { VideoWidget } from "../components/VideoWidget";
 import { Footer } from "../components/Footer";
+import { Header } from "../components/Header";
 
 export function Home() {
 	return (
 		<div>
 			<header>
+				<Header />
```

Comment:
Would prefer the header to be part of the menubar so that it is reusable but we can always refactor this